src/components/ui/Chat/ChannelList/ChannelItem.tsx

```diff
@@ -1,24 +1,26 @@
 // Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-import React, { HTMLAttributes } from 'react';
+import React, { HTMLAttributes, ReactNode } from 'react';
 
 import { StyledChannelItem } from './Styled';
 import { useChannelListContext } from './ChannelListContext';
 import PopOver from '../../PopOver';
-import PopOverItem, { PopOverItemProps } from '../../PopOver/PopOverItem';
 import Button from '../../Button';
 import { BaseProps } from '../../Base';
 import { Dots } from '../../icons';
 
 export interface ChannelItemProps extends Omit<HTMLAttributes<HTMLLIElement & HTMLLIElement>, 'css'>, BaseProps {
+  /* Display name of the channel */
   name: string;
+  /* Unique identifier of the channel */
   arn: string;
```

Comment:
is `arn` term valid in context of this component? should we have more generic variable name here?

Reply:
I wasn't sure about this. Probably not, we should just use id, I just personally found it difficult to keep id vs. arn straight as they were changing it mid-development for the POC.